beacon-chain/core/transition/state.go

```diff
@@ -80,7 +79,7 @@ func OptimizedGenesisBeaconState(genesisTime uint64, preState state.BeaconState,
 		return nil, errors.New("no eth1data provided for genesis state")
 	}
 
-	randaoMixes := customtypes.RandaoMixes{}
+	var randaoMixes [65536][32]byte
```

Comment:
Exported getters/setters use a plain array. This is because these custom types were never expected to be used outside of the `state` package. The plan was to use them internally for deduplication. Maybe this can be changed in the future, but I'm keeping it simple for now.